www/rustup.css

```diff
@@ -165,7 +165,7 @@ hr {
     position: relative;
     width: fit-content;
     height: fit-content;
-    top: 50%;
+    top: 27px;
```

Comment:
I'm not very familiar with CSS, so please correct me if I'm wrong, but I believe `top: 50%` with `-webkit-top: 27px` would be a better option here?

Reply:
![image](https://github.com/rust-lang/rustup/assets/23432548/2b8a67fc-b662-48da-8068-778c0f437e6a)
Good idea, but it seems that `top` takes precedence.

Reply:
@noirgif Just did some extra research and it turns out the ordering is important (`top` should come after `-webkit-top`): https://stackoverflow.com/a/7080674

Reply:
There doesn't seem to be a CSS extension called `-webkit-top`: https://developer.mozilla.org/en-US/docs/Web/CSS/Webkit_Extensions